SonarQube.Client.Tests/SonarQubeService_GetRulesAsync.cs

```diff
@@ -0,0 +1,219 @@
+﻿/*
+* SonarQube Client
+* Copyright (C) 2016-2018 SonarSource SA
```

Comment:
Should probably be "2016-2019". Still, almost all of the existing files need to be changed so I'll bulk update them  later.